frontend/src/modules/sampleEmibJson.js

```diff
@@ -1,65 +1,72 @@
-    },
-    {
-      id: 1,
-      to: "Claude Huard (Manager, Quality Assurance Team)",
-      from: "Marina Richter (Quality Control Analyst, Quality Assurance Team)",
-      subject: "Informal Training on Serv",
-      date: "Friday, November 4",
-      body:
-        "Hello Claude,\n\nDuring our last meeting, Danny had mentioned that he learned a lot about the Serv system during the pilot testing exercise with the IT unit.  While talking to other team members, some mentioned they were trained on and worked with an older version of Serv in previous jobs. However, there are a few of us who have never used it. I would like to know if there would be opportunities to be trained on Serv?\n\nMarina"
-    },
-    {
-      id: 2,
-      to: "Claude Huard (Manager, Quality Assurance Team)",
-      from: "Charlie Wang (Quality Control Analyst, Quality Assurance Team)",
-      subject: "Report deadline",
-      date: "Friday, November 4",
-      body:
-        "Hello Claude,\n\nI am working with Clara Farewell from the Research and Innovations unit on evaluating the quality of a training approach and I am having a hard time getting a hold of her. I am starting to be concerned because I have been waiting on her part of the work to complete the evaluation report.\nFor the past three weeks, we had scheduled working meetings on Friday afternoons and although she did cancel the first one, she was absent the past two, without notice. She did not answer my attempts to contact her by phone or email. I am worried that I will not be able to complete the report by the end of next Friday without her content.\n\nOn another note, I was told by one of my colleagues from the Program Development unit that his director, Bartosz Greco, would invite employees from other units to help them develop a new training program. They want to take a multiple perspectives approach. I’m very much interested in participating in this process. As usual, manager permission is required for participation. I am wondering what you think?\n\nThank you,\nCharlie"
+  test_internal_name: "emibSampleTest",
+  questions: {
```

Comment:
Some of these layers feel unnecessary to me. Why wouldn't you just have:

questions: {
 en: [],
 fr: []
}

The email array seems redundant and misrepresents the information - why not "emails" since it represents an array?